entries/QUnit.config.xml

```diff
@@ -40,6 +40,11 @@
 				This property allows QUnit to run specific modules identified by the hashed version of their module name. You can specify one or multiple modules to run.
 			</desc>
 		</property>
+		<property name="noglobals" type="Boolean" default="false">
+			<desc>
+				<p>Enabling this flag will cause QUnit to check if any new properties have been added to the global context after each test.</p>
```

Comment:
This should document that with this flag enabled, adding new global properties will generate test failures.